org.nongnu.gsequencer.gsequencer.json

```diff
@@ -212,15 +254,32 @@
                 }
             },
             "post-install": [
+                "mkdir -p /app/share/doc/gsequencer/pdf/",
+                "install -c -p -m 644 pdf/user-manual-a4.pdf /app/share/doc/gsequencer/pdf/user-manual-a4.pdf",
+                "install -c -p -m 644 pdf/user-manual-letter.pdf /app/share/doc/gsequencer/pdf/user-manual-letter.pdf",
```

Comment:
if you use `install -D` you don't need the mkdir. It's usually good practice to do so

Reply:
Just did it so.